scipy/signal/_backend.py

```diff
@@ -0,0 +1,80 @@
+import scipy._lib.uarray as ua
+from scipy.signal import _api
+import numpy as np
+
+__all__ = [
+    'register_backend', 'set_backend',
+    'set_global_backend', 'skip_backend'
+]
+
+
+class _ScipySignalBackend:
+    __ua_domain__ = "numpy.scipy.signal"
+
+
+    @staticmethod
+    def __ua_function__(method, args, kwargs):
+        fn = getattr(_api, method.__name__, None)
+
+        if fn is None:
+            raise NotImplementedError
+            return
+        return fn(*args, **kwargs)
+
+
+    # @ua.wrap_single_convertor
+    # def __ua_convert__(value, dispatch_type, coerce):
+    #     if value is None:
```

Comment:
I need to look more into it. Removing the comments leads to some test failures at present.